### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by KevinTech0916** [AbdulMoqeet/HRAN#2]
How many epochs and time on training x2 x3 x4 x8?
Will you plan to provide pre-trained models?
Thank you!!

**Comment by AbdulMoqeet** [comment]
I trained for 1000 epochs but the accuracy improvement after 300-400 epochs is minimal.

Yes, I will provide the pre-trained models soon.

**Comment by AbdulMoqeet** [comment]
The pre_trained models are upload on [Google Drive.](http://bit.ly/31GiwRX)
